DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Administrator/_SearchableAdminCard.cshtml

```diff
@@ -53,9 +53,16 @@
           Manage roles
         </a>
       }
-      <a class="nhsuk-button delete-button nhsuk-u-margin-bottom-1" role="button">
-        Deactivate admin account
-      </a>
+      @if (!Model.IsCentreManager) { 

```

Comment:
Just to confirm, nobody can deactivate a centre manager's account? Do we check for that elsewhere? (e.g. in the controller, do we redirect if the admin id is for a centre manager)

Reply:
This was suggested on Show tell meeting that Centre Manager should not be able to deactivate the admin account

Reply:
Ah, so is this supposed to be stopping centre managers from seeing the deactivate button? Because what I _think_ this is doing is not showing the deactivate button on the cards for admin users that are centre managers.

Reply:
![image](https://user-images.githubusercontent.com/90783892/140545694-9afa309d-ee60-4ea0-b7e5-8f1be6c32619.png)

Reply:
Yes, that's correct. I have added a slacker conversation about this above.

Reply:
Confirmed with Steve on slack this is the correct behaviour 👍